docs/content/preview/tutorials/_index.md

```diff
@@ -3,64 +3,74 @@ title: Tutorials for developing with YugabyteDB
 headerTitle: Tutorials
 linkTitle: Tutorials
 description: How to develop applications with YugabyteDB using your preferred environment, software and programming language.
-headcontent: Step-by-step guides for building scalable and fault tolerant applications with YugabyteDB using your favorite programming language, software, and services
+headcontent: Step-by-step guides for building scalable and fault tolerant applications with YugabyteDB using your favorite programming language, services, and frameworks
 type: indexpage
 cascade:
   unversioned: true
 ---
 
-### First applications
+{{< sections/2-boxes >}}
+  {{< sections/bottom-image-box
+    title="Create a hello world application"
+    description="Build the simplest application in your favorite language using YugabyteDB as a database."
```

Comment:
```suggestion
    description="Build a basic application in your favorite language using YugabyteDB as a database."
```

Reply:
Not sure if we have to strongly follow the usage of words like simple and easy.. but if we want to.. then go with my suggestion.